packages/core/src/app/ui/form/FormField.tsx

```diff
@@ -26,18 +27,25 @@ const FormField: FunctionComponent<FormFieldProps> = ({
     input,
     name,
     id,
+    useFloatingLabel,
 }) => {
     const renderField = useCallback(
         (props) => (
             <>
+                {useFloatingLabel && input(props)}
+
                 {label && (typeof label === 'function' ? label(name) : label)}
                 {labelContent && !label && (
-                    <Label htmlFor={name} id={`${id ?? name}-label`}>
+                    <Label
+                        htmlFor={name}
+                        id={`${id ?? name}-label`}
+                        useFloatingLabel={useFloatingLabel}
+                    >
                         {labelContent}
                     </Label>
                 )}
 
-                {input(props)}
+                {!useFloatingLabel && input(props)}
```

Comment:
The entire floating label implementation is based on the `+` CSS selector, which is used to apply floating styles to the label that follows a text field.

Because there is no CSS selector to do the opposite of `+` (see [stackoverflow](https://stackoverflow.com/questions/15449622/opposite-of-adjacent-sibling-selector)), we must change the placement of `<Label/>` according to the state of A/B testing.